package.json

```diff
@@ -13,8 +13,10 @@
   "keywords": [],
   "type": "module",
   "dependencies": {
+    "@iarna/toml": "^2.2.5",
```

Comment:
Could you please remove this library since I'm assuming it's not used anymore and push the code again?

Reply:
Solved in commit [09791bfc](https://github.com/sdthaker/Node-TILify/pull/13/commits/09791bfc36a7c5b601649075c18c845a87c17d7c)